documents/Specification-Format-Style-Guidelines.md

```diff
@@ -0,0 +1,296 @@
+# Protocol Buffer Style Guidelines
+
+## Style Guidelines for "specification" comments
+- Wrapping is 80 characters for top-level messages.
+- Wrap everything else to 80 characters or 100, depending
+   - Prefer to wrap lines to 80 characters
+   - It is OK to go over 80, but less than 100, if it makes the
+     text clearer and easier to read.
+     - This SHOULD be rare, however.
+   - If you have links or tags, wrapping at 100 characters for clarity
+     is acceptable.
+   - We choose 80 characters here to work better in reviews because Github
+     comment boxes are limited to roughly 80 characters width. The primary
+     "code" interaction with proto files and markdown files is in review.
+     Also, some markdown processors struggle with longer lines.
+- Use blank lines (with `*` prefix) for paragraph breaks in message or
```

Comment:
I think might be a little confusing. When should a message have a blank line and when should it have a <p> ?

Reply:
All specification content should be as close to clean markdown as the current processor permits.  The use of `<p>` in _fields_ is necessitated by limitations of markdown tables; messages should _always_ prefer a blank line over a `<p>` tag.

I'll add some clarifying text here.

Reply:
Updated.